cmd/karavictl/cmd/role_create_test.go

```diff
@@ -58,7 +58,7 @@ func TestRoleCreateHandler(t *testing.T) {
 
 		cmd := NewRootCmd()
 		cmd.SetOutput(&gotOutput)
-		cmd.SetArgs([]string{"--admin-token", "admin.yaml", "role", "create", "--addr", "https://role-service.com", "--insecure", "--role=bar=powerflex=11e4e7d35817bd0f=mypool=75GB"})
+		cmd.SetArgs([]string{"role", "create", "--addr", "https://role-service.com", "--insecure", "--role=bar=powerflex=11e4e7d35817bd0f=mypool=75GB", "--admin-token", "admin.yaml", "--addr", "proxy.com"})
```

Comment:
This seems to have two --addr flag in one command is it supposed to be overwritten by the second one "proxy.com"?

Reply:
Removed duplicate address flags.